### PR TITLE
Moving metadata.json to typescript

### PR DESCRIPTION
**Pull request by JoshVarga**
Moving metadata.json to typescript, and removing the metadata 'window' variables

### CODE REVIEW

File: src/components/DialogBox/CombinationalAnalysis.vue [hunk 1]
```diff
@@ -199,7 +199,7 @@ function createLogicTable() {
         SimulatorState.dialogBox.combinationalanalysis_dialog = false
         output.value = []
         solveBooleanFunction(booleanInputVariables, booleanExpression)
-        if (output != null) {
+        if (output.value != null) {
```
Comment (tachyons) [inline src/components/DialogBox/CombinationalAnalysis.vue:202]: This change is related?

Comment (JoshVarga) [reply src/components/DialogBox/CombinationalAnalysis.vue]: Unrelated so removed.

**Comment by gr455** [comment]
@niladrix719 For your review

**Review by gr455 — APPROVED** [review]

File: src/components/Panels/ElementsPanel/ElementsPanel.vue [hunk 2]
```diff
@@ -237,4 +236,4 @@ function getTooltipText(elementName) {
 .v-expansion-panel-title {
     min-height: 36px;
 }
-</style>
+</style>#/simulator/src/metadata
```
Comment (niladrix719) [inline src/components/Panels/ElementsPanel/ElementsPanel.vue:239]: this can be removed

**Review by niladrix719 — COMMENTED** [review]
Looks Good!

File: src/components/Panels/ElementsPanel/ElementsPanel.vue (hunk 2 above)
Comment (JoshVarga) [reply src/components/Panels/ElementsPanel/ElementsPanel.vue]: Done

**Pull request merged**